Changelog.md

```diff
@@ -1,4 +1,5 @@
 # Next
+- Added tests for Single+Response and refactored Shared Test Helpers.
```

Comment:
I think moving it to the bottom would be great. Right under where we mention adding `Single<Response>`.